global-privacy-control/README.md

```diff
@@ -10,6 +10,7 @@ This set of tests verifies implementation of Global Privacy Control signal. In p
 - that `navigator.globalPrivacyControl` API is available in all frames
 - that the right thing happens when user opts out of GPC and
 - that excluded domains, as defined in the privacy remote configuration, are taken into account.
+- note: On platforms where header modifications is limited headers should only be modified on sites listed in gpcHeaderEnabledSites inside the settings of the gpc feature of the remote config.
```

Comment:
I think we should convert this to sth like https://github.com/duckduckgo/privacy-reference-tests/tree/main/block-third-party-tracking-cookies#platform-exceptions . This is a standard way to document any use of `exceptPlatforms`.